lms/djangoapps/teams/views.py

```diff
@@ -368,6 +368,9 @@ def post(self, request):
             }, status=status.HTTP_400_BAD_REQUEST)
         else:
             team = serializer.save()
+            if not has_access(request.user, 'staff', course_key):
```

Comment:
Does has_access(request.user, 'staff') return True for all these variations?

1) Global users with is_staff permissions (looks like there is a test covering this)
2) Course staff (there is a test covering this)
3) Discussion Admin/Moderator/TA (I'm not sure any of these are restricted to course staff)

Perhaps @explorerleslie does not care if #3 is not covered, in cases where Discussion Admin/Moderator/TA are not course staff members. I'm not sure how closely we should link Forums roles to the more general notion of Teams (as opposed to governance of the discussion topic within each team).